backend/src/common/decorator/extract-user-id.decorator.ts

```diff
@@ -10,11 +9,8 @@ import { createParamDecorator, ExecutionContext } from '@nestjs/common';
 export const ExtractUserId = createParamDecorator((data: unknown, ctx: ExecutionContext): number => {
   const request = ctx.switchToHttp().getRequest();
 
-  // 개발 환경에서는 headers에 있는 x-my-id를 userId로 사용
-  if (process.env.NODE_ENV === 'development') {
-    return +request.headers['x-my-id'];
+  if (request.user === undefined || request.user.userId === undefined) {
```

Comment:
```suggestion
  if (request.user?.userId === undefined) {
```
같은 의미인데 해당 코드 제안드립니다!

Reply:
오... 좋아요 이렇게 할 수 있군요..!

Reply:
수정했습니다..!

Reply:
진짜 타스 멍미 댑악